### ISSUE TITLE
Auto exposure is not working

### ISSUE THREAD

**Issue by ghost** [libuvc/libuvc#132]
Seems auto exposure is not working. Anyone tried that?

**Comment by dmikushin** [comment]
Yes, same for me (very dark). It's actually even more interesting. I've opened MacOS' builtin Photo Booth before and after launching camera from within libuvc. First time Photo Booth video was bright "as usual". But when I opened Photo Booth *after* testing libuvc, video became a lot darker. And there are no settings to reset it back. Congratulations @ktossell hijacking camera so brutally, that even MacOS can't stand it :+1:

**Comment by dmikushin** [comment]
Sorry, actually auto exposure works. @deeprun please make sure you supply value of 2:

```
const uint8_t UVC_AUTO_EXPOSURE_MODE_AUTO = 2;
res = uvc_set_ae_mode(devh, UVC_AUTO_EXPOSURE_MODE_AUTO);
```

**Comment by ktossell** [comment]
I've posted a PR that applies this correction to the example code; does this look all right? https://github.com/libuvc/libuvc/pull/138

**Comment by dmikushin** [comment]
@ktossell thank you for looking. Right. Yet, as mentioned in #118 auto exposure won't get through in Linux. Still to be investigated on that side.
